source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+using MathNet.Numerics.LinearAlgebra.Double;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            printHeader("This example shows conversions to/from Transformation Matrix");
+
+            // Read a transformation matrix
+            var transformationMatrixFile = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)
+                               + "/Zivid/RobotTransform.yaml";
+            Console.WriteLine("Getting PoseState:");
+            var transformationMatrix = readTransform(transformationMatrixFile);
+            var mathMatrix = zividToMathDotNet(transformationMatrix);
+            Console.WriteLine(matrixToString(mathMatrix));
+
+            // Extract rotation matrix and translation vector
+            var rotationMatrix = mathMatrix.SubMatrix(0, 3, 0, 3);
+            var translationVector = mathMatrix.SubMatrix(0, 3, 3, 1);
+
+            Console.WriteLine("RotationMatrix:\n" + matrixToString(rotationMatrix));
+            Console.WriteLine("TranslationVector:\n" + matrixToString(translationVector.Transpose()));
+
+            /*
+             * Convert from Rotation Matrix (Zivid) to other representations of orientation (Robot)
+             */
+            printHeader("Convert from Zivid (Rotation Matrix) to Robot");
+            var axisAngle = rotationMatrixToAngleAxis(rotationMatrix);
+            Console.WriteLine("AxisAngle:\n" + matrixToString(axisAngle.Item1.Transpose()) + ", " + String.Format(" {0:G4} ", axisAngle.Item2));
+            var rotationVector = axisAngle.Item1 * axisAngle.Item2;
+            Console.WriteLine("Rotation Vector:\n" + matrixToString(rotationVector.Transpose()));
+            var quaternion = rotationMatrixToQuaternion(rotationMatrix);
+            Console.WriteLine("Quaternion:\n" + matrixToString(quaternion.Transpose()));
+            var rpyList = rotationMatrixToRollPitchYawList(rotationMatrix);
+
+            /*
+             * Convert to Rotation Matrix (Zivid) from other representations of orientation (Robot)
+             */
+            printHeader("Convert from Robot to Zivid (Rotation matrix)");
+            var rotationMatrixFromAxisAngle = axisAngleToRotationMatrix(axisAngle);
+            Console.WriteLine("Rotation Matrix from Axis Angle:\n" + matrixToString(rotationMatrixFromAxisAngle));
+
+            var rotationMatrixFromRotationVector = rotationVectorToRotationMatrix(rotationVector);
+            Console.WriteLine("Rotation Matrix from Rotation Vector:\n" + matrixToString(rotationMatrixFromRotationVector));
+            var rotationMatrixFromQuaternion = quaternionToRotationMatrix(quaternion);
+            Console.WriteLine("Rotation Matrix from Quaternion:\n" + matrixToString(rotationMatrixFromQuaternion));
+
+            rollPitchYawListToRotationMatrix(rpyList);
+
+
+            Console.WriteLine("Rotation from print:\n" + matrixToString(rotationMatrix));
```

Comment:
You are using rotationMatrix created in line 32. If you look at the C++ sample, the point is to use rotationMatrixFromQuaternion to show that the conversion works well the other way (from rot mat to quat and then back to rot mat).
Implement here equivalent of https://github.com/zivid/zivid-cpp-samples/blob/d840629c50b18d6428ae6749774497f310980dbf/source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp#L248-L251

Reply:
Done